frontend/src/App.tsx

```diff
@@ -53,6 +53,9 @@ const router = createBrowserRouter(
           element={<ProjectObject projectType="investointihanke" />}
         />
         <Route path="kohde/uusi" element={<ProjectObject projectType="investointihanke" />} />
+        <Route path="kunnossapitohanke/luo" element={<MaintenanceProject />} />
+        <Route path="kunnossapitohanke/:projectId" element={<MaintenanceProject />} />
+        <Route path="kunnossapitohanke/:projectId?tab=:tabView" element={<MaintenanceProject />} />
```

Comment:
IIRC query parameters don't get handled by `react-router-dom`. In every route there could be any query parameters, and they can be read using `useSearchParams`. So, this route shouldn't be needed here.

An alternative way would be to use something like `kunnossapitohanke/:projectId/:tabView`, if you wanted to include the selected tab in the route.